tests/integration_tests/tests/agentless_tests/test_resume.py

```diff
@@ -335,6 +335,7 @@ def test_force_resume_cancelled_nonresumable(self):
         self.assertEqual(execution.status, 'cancelled')
 
         self._unlock_operation('interface1.op_nonresumable')
+        time.sleep(5)
```

Comment:
well it's just for we will not see the unauthorized error in the log?

Reply:
pretty much

we note that also one test already had a sleep after unlock_operation, and tbh every one should do that, so unlock_operation itself should do the sleep